rskj-core/src/main/java/co/rsk/rpc/modules/debug/DebugModuleImpl.java

```diff
@@ -143,6 +143,16 @@ private JsonNode traceBlock(Block block, TraceOptions options) {
         return programTraceProcessor.getProgramTracesAsJsonNode(txHashes);
     }
 
+    private boolean testFunctionNotCovered() {
+        boolean var1 = false;
+        boolean var2 = true;
+        boolean var3 = false;
+        if (!var1 && var2 || !var3) {
+            return true;
+        }
+        return false;
+    }
+
```

Comment:
question:

What is the meaning for each of this var? Can you explain it? What do you think in rename it to a more explanatory one?

Reply:
Ah! this code will be removed, it was only to test if the cloud would report the missing tests. I'll remove this as soon as we decide if we're ok with the changes. 💪🏼

Reply:
@nagarev , it seems like it's working... do you think we could remove it now and get the final version that can be merged to master?

Reply:
In that case, I am approving it, since this code will be removed.

Reply:
Test function was removed. 💪🏼